docs/user/production-considerations/task-manager-health-monitoring.asciidoc

```diff
@@ -71,6 +75,21 @@ logging:
 
 These stats are logged based the number of milliseconds set in your <<task-manager-settings,`xpack.task_manager.poll_interval`>> setting, which means it could add substantial noise to your logs. Only enable this level of logging temporarily.
 
+==== Automatic logging
+
+By default, the health API runs at a regular cadence and each time it runs, it will attempts to self evaluate its performance. If this self evaluation yields a potential problem,
```

Comment:
```suggestion
By default, the health API runs at a regular cadence and each time it runs, it will attempt to self evaluate its performance. If this self evaluation yields a potential problem,
```